Foundation/include/Poco/Any.h

```diff
@@ -28,6 +28,8 @@
 
 namespace Poco {
 
+#define POCO_SMALL_OBJECT_SIZE 64
```

Comment:
this is defined in the Config.h, not needed here

Reply:
I am not experienced with c++.  The compiler generated an error `out of scope`, I am merely fixing the problem.